packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -539,7 +554,7 @@ async function handler(req: NextApiRequest & { userId?: number | undefined }) {
 
   const teamMembers = await Promise.all(teamMemberPromises);
 
-  const attendeesList = [...invitee, ...guests, ...teamMembers];
```

Comment:
This is causing collective events to not add all team members to the created calendar event.

Reply:
In #7756 team members are added to the `attendeesList` on line 1442 before creating the booking in the DB & sending it to the event manager to create the calendar event. 

I tested this locally and the db record, email, and calendar event description have the team members included.

Reply:
Yeah, that data is not the same one being sent to the EventManager. It is passed directly to the DB only:

<img width="210" alt="image" src="https://user-images.githubusercontent.com/3504472/226426572-62b8e5ef-871d-481a-a86f-5750f897cf38.png">